dhis-2/dhis-api/src/main/java/org/hisp/dhis/feedback/ErrorCode.java

```diff
@@ -245,6 +245,8 @@
 
     /* Metadata Validation (continued) */
     E4060( "Object could not be deleted: {0}" ),
+    E4061(
+        "DashboardItem `{0}` has invalid reference `{1}` with id `{2}`. You might not have READ permission to this object." ),
```

Comment:
To make it more consistent with existing error codes use something like:

"DashboardItem `{0}` object reference `{1}` with id `{2}` not found or not accessible"

Reply:
fixed